bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/ThingTypeRegistry.java

```diff
@@ -151,7 +151,11 @@ public ChannelType getChannelType(ChannelUID channelUID) {
      */
     public ChannelType getChannelType(ChannelUID channelUID, Locale locale) {
         ThingType thingType = this.getThingType(channelUID.getThingTypeUID(), locale);
-        return thingType != null ? thingType.getChannelType(channelUID) : null;
+        if (thingType != null) {
+            ChannelTypeUID channelTypeUID = thingType.getChannelTypeUID(channelUID);
+            return TypeResolver.resolve(channelTypeUID);
```

Comment:
Isn´t it enough that the caller get´s null? If we assume there will be bindings that don´t define channel types, there will be a lot of log messages like these. What severity should the log message have?